### DISCUSSION TITLE
QREstatico com erro no Itau

### DISCUSSION THREAD

**Discussion by flaviolenz** [bacen/pix-api#396] [General]
Alguem tem uma conta no Itau e pode testar pra mim um Pix Copia/Cola ?
(nao pagar - eh so ver se ta pegando os dados do DICT)

00020126580014br.gov.bcb.pix01365a5ea8c2-cbb5-4c25-a7be-2a778871fc5f5204000053039865406650.005802BR5908SoftLenz6008Brasilia62290525e1v169u155f1m1232xBrookly63045BD2

**Comment by renatofrota** [comment]
Consegui ler no Itaú, sem problemas (tanto direto pelo "Copia e Cola" quanto por um QR Code gerado a partir dessa string).
